tests/php/Configuration/Parser/MenuParserTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Bolt\Tests\Configuration\Parser;
+
+use Bolt\Configuration\Parser\MenuParser;
+use PHPUnit\Framework\TestCase;
+use Tightenco\Collect\Support\Collection;
+
+class MenuParserTest extends TestCase
+{
+    /** @var MenuParser */
+    private $menuParser;
+
+    protected function setUp(): void
+    {
+        $this->menuParser = new MenuParser();
```

Comment:
new instance should be created in each test separately to avoid possible side effects

Reply:
Done